ui/ducks/metamask/metamask.js

```diff
@@ -272,8 +273,10 @@ export function getNftsDropdownState(state) {
 
 export const getNfts = (state) => {
```

Comment:
`getNftContracts` may need a similar fix?

Reply:
that'r right, fixed 🙏